### ISSUE TITLE
Flickr Your Groups photos

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yendo/gphotoframe#54]
```
Random pickup a group from "Your Groups" in Flickr.
Show photos from the group.
```

Original issue reported on code.google.com by `yendo0206` on 13 May 2010 at 8:09

**Comment by GoogleCodeExporter** [comment]
Original comment by `yendo0206` on 13 May 2010 at 8:09
- Added labels: **Component-Plungins**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in revision 3e52e3bdc6.
```

Original comment by `yendo0206` on 13 May 2010 at 10:27
- Changed state: **Fixed**
